pkg/yurthub/proxy/remote/remote.go

```diff
@@ -80,6 +89,15 @@ func (rp *RemoteProxy) Name() string {
 }
 
 func (rp *RemoteProxy) ServeHTTP(rw http.ResponseWriter, req *http.Request) {
+	rp.reverseProxy.Transport = rp.currentTransport
+	auth := strings.TrimSpace(req.Header.Get("Authorization"))
+	if auth != "" {
```

Comment:
Can you add a comment here to explain when the auth is nil and when it is not nil?

Reply:
comments were added.